### DISCUSSION TITLE
Dynamic Land cover data

### DISCUSSION THREAD

**Discussion by JeromeJeitany** [iiasa/CWatM#51] [Q&A]
I am currently using the CWatM using specific basin nc and raster files to test it at a lower resolution. 
Would it be possible to use a finer land cover map while providing all required information for each class specified (crop coefficient,Intercept,....) ? 
by doing so, should I also change the soil and routing maps?
Thank you for the help,
Jerome

**Comment by mikhailsmilovic** [comment]
Guten Tag, 

If one simply wants to improve the land class input maps and keep the 5 arcminutes resolution, one could start by updating only the land class cover. From here, one could also improve the crop coefficient, intercept, etc., but there is no explicit need. There is no explicit need to update the soil and routing maps. 

Input maps must be at the same resolution, except for climate data. 
For a first step, the finer (meaning, here, higher resolution) maps could be used to create improved lower-resolution maps. For example, we provide the global data at 5 arcminutes (~10km), and any of these maps can be replaced and improved.

If the interest is to simulate at a higher resolution, all maps, such as soil and routing, must be changed.
